src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx

```diff
@@ -4,201 +4,16 @@ tags:
   - Integrations
   - Open source telemetry integrations
   - OpenTelemetry
-metaDescription: New Relic offers an integration that sends your OpenTelemetry data to your New Relic account.
+metaDescription: New Relic supports sending OpenTelemetry data to your New Relic account.
 redirects:
   - /docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/new-relics-opentelemetry-integration
   - /docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/opentelemetry-exporter
   - /docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-exporter
   - /docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry
 ---
 
-You can send OpenTelemetry data to New Relic and analyze the results in our observability platform. This introduction can help you decide if you want to get started with OpenTelemetry. If you're already familiar with OpenTelemetry and want to jump into the setup, go to our [OpenTelemetry quick start](https://docs.newrelic.com/docs/integrations/open-source-telemetry-integrations/opentelemetry/set-opentelemetry-new-relic).
+TODO: Need outline of here
```

Comment:
@rhetoric101 Interested in your ideas of how to structure this intro page to guide users to the other pages.